LISA/Executor.cpp

```diff
@@ -184,20 +184,20 @@ uint32_t Executor::GetStorageDetails(const std::string& type,
         details.persistentUsedKB = std::to_string(fs::getDirectorySpace(fs::getAppsStorageDir()));
     } else {
         INFO("Calculating usage for: type = ", type, " id = ", id, " version = ", version);
-        std::vector<std::string> appsPaths = storage->GetAppsPaths(type, id, version);
```

Comment:
If I'm not mistaken, you can remove "storage" function parameter and "ds" member from LISAImplementation.cpp since they will be no longer used.

Reply:
True, but I did not want to do that yet because I don't know what @mariusz-sikora-red intentions were for this storage parameter in the future. But if @mariusz-sikora-red agrees, I can clean that up. No problem.

Reply:
Yes, storage function parameter was used before DataStorage was moved to Executor. It is not needed  now and Executor::dataBase member can be used. However the idea was to allow it to be testable, so maybe in the future when UT will be introduced we could pass DataStorage implementation/mock to Executor::Configure method instead of creating it inside.

If this is not a problem, you could clean it up please as it is not needed anymore and it will be a good match with the commit message.

Reply:
@szymon-polke-red  @mariusz-sikora-red , Done.